docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+      "Given label: ORG, predicted label: O\n",
+      "\n",
+      "14. \u001b[31mCan\u001b[0m/ U.s. Dollar Exchange Rate: 1.3570\n",
+      "Given label: LOC, predicted label: O\n",
+      "\n",
+      "15. \u001b[31mBorn\u001b[0m in 1937 in the central province of Anhui, Dai came to Shanghai as a student and remained in the city as a prolific author and teacher of Chinese.\n",
+      "Given label: LOC, predicted label: O\n",
+      "\n",
+      "16. 1990 - British historian Alan John Percivale \u001b[31m(\u001b[0mA.j.p.) Taylor died.\n",
+      "Given label: PER, predicted label: O\n",
+      "\n",
+      "17. \u001b[31mPace\u001b[0m bowler Ian Harvey claimed three for 81 for Victoria.\n",
+      "Given label: PER, predicted label: O\n",
+      "\n",
+      "18. But one must not forget that the Osce only has limited powers there,\" said \u001b[31mCotti\u001b[0m, who is also the Swiss foreign minister.\"\n",
+      "Given label: O, predicted label: PER\n",
+      "\n",
+      "19. Specter met Crown Prince Abdullah and Minister of Defence and Aviation Prince \u001b[31mSultan\u001b[0m in Jeddah, Saudi state television and the official Saudi Press Agency reported.\n",
+      "Given label: PER, predicted label: O\n",
+      "\n",
+      "20. \u001b[31mSporting\u001b[0m his customary bright green outfit, the U.s. champion clocked 10.03 seconds despite damp conditions to take the scalp of Canada's reigning Olympic champion Donovan Bailey, 1992 champion Linford Christie of Britain and American 1984 and 1988 champion Carl Lewis.\n",
+      "Given label: ORG, predicted label: O\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "show_issues(issues, labels, pred_probs, given_words, sentences, exclude=[(0, 1), (1, 0)], top=20) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96d04902",
+   "metadata": {},
+   "source": [
+    "More than half of the potential label issues are identified correctly. As shown above, some examples are ambigious and require manual checking. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d213b2b2",
+   "metadata": {},
+   "source": [
+    "## 4. Most common word-level token mislabels \n",
+    "\n",
+    "It may be useful to examine the most common word-level token mislabels. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "e4a006bd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'/' is mislabeled 42 times\n",
+      "-----------------------------\n",
+      "labeled as O, but predicted as LOC 36 times\n",
+      "labeled as O, but predicted as PER 4 times\n",
+      "labeled as O, but predicted as ORG 2 times\n",
```

Comment:
I'm not sure this info is super useful without showing at least 1 example of a sentence with each type of issue.
So let's call `show_sentence_issues()` inside of this function to show just one example sentence with each type of issue, after reporting how many times this type of issue occurred in the dataset.